flake.nix

```diff
@@ -1,9 +1,53 @@
 {
+  description = "A Text User Interface (TUI) designed for interacting with the Jujutsu version control system";
+
   inputs = {
-    systems.url = "github:nix-systems/default";
-    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
     flake-parts.url = "github:hercules-ci/flake-parts";
+
+    flake-compat = {
+      url = "github:NixOS/flake-compat";
+      flake = false;
+    };
   };
 
-  outputs = inputs: inputs.flake-parts.lib.mkFlake { inherit inputs; } ./nix;
+  outputs =
+    inputs@{
+      self,
+      flake-parts,
+      ...
+    }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
```

Comment:
can you make this a oneliner that just imports a module from ./nix ? like:

`inputs: inputs.flake-parts.lib.mkFlake { inherit inputs; } ./nix` 

lets keep flake.nix as a manifest and have a main flake-parts entrypoint at ./nix.